terminus-ui/selection-list/src/selection-list.component.ts

```diff
@@ -927,4 +939,22 @@ export class TsSelectionListComponent implements
     this.inputElement.nativeElement.value = '';
   }
 
+  /**
+   * Scroll the active option into view
+   */
+  private scrollActiveOptionIntoView(): void {
+    const ctrlValue = this.selectionListFormControl.value || [];
+    const allOptions = this.trigger.selectionListPanel.options.toArray().map(o => o.value);
+    const index = allOptions.findIndex(o => this.valueComparator(o, ctrlValue[0]));
+    this.trigger.selectionListPanel.scrollTop = getOptionScrollPosition(
+      index,
+      this.trigger.itemHeight,
+      this.trigger.selectionListPanel.scrollTop,
+      SELECTION_LIST_PANEL_MAX_HEIGHT,
+    );
+    // Only update the focus when it's not autocomplete
+    if (!this.allowUserInput) {
+      this.trigger.selectionListPanel.keyManager.setActiveItem(index);
+    }
```

Comment:
Doesn't look like this is tested:

![Screen Shot 2020-01-16 at 10 28 14 AM](https://user-images.githubusercontent.com/270193/72537970-0ffa8380-384b-11ea-98f8-dacecb8e54b6.png)